workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/views/static_report.py

```diff
@@ -248,11 +226,29 @@ def zonal_mean_hovmoller_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
 @hovmoller_plot_manager.register
 def zonal_mean_hovmoller_bias_plots(diagnostics: Iterable[xr.Dataset]) -> HVPlot:
     return plot_2d_matplotlib(
+        diagnostics, "zonal_mean_bias", dims=["time", "latitude"], cmap="RdBu_r",
+    )
+
+
+def time_mean_map_plots(
```

Comment:
Again "map_plots" vs "plot_cube". You should settle on one of "cube/map/horizontal" and use it consistently.